src/TerminalObject/Basic/BasicTerminalObject.php

```diff
@@ -18,7 +18,7 @@ abstract class BasicTerminalObject implements BasicTerminalObjectInterface
      */
     protected function set($key, $value)
     {
-        if (strlen($value)) {
+        if ($value !== '') {
```

Comment:
Doesn't this change the treatment of `null`? Previously it wouldn't change the properly, but now it will set it to `null`

Reply:
fixed